understanding/22/target-size-minimum.html

```diff
@@ -94,25 +96,11 @@ <h3>Spacing versus size</h3>
-          <figcaption>Three illustrations showing the offsets between two targets when they abut and overlap.</figcaption>
-      </figure>
-
-        <figure id="pointer-target-inline-links">
-            <img src="img/pointer-target-example6-inline-links.png" alt="Inline links in a block of text and in an ordered list (see legend below)" />
-          <figcaption>Two examples show inline text links. The first example has various links in a block of text. The font size is 14px, line height is 16.8px. Depending on the position of links in the text, the vertical offset of links is smaller than 24 CSS pixels. For linked footnote numbers, the horizontal offset is also below 24 CSS pixels. The inline exception allows inline targets to have an offset below the requirement of 24 by 24 CSS pixels.</figcaption>
-      </figure>
-
-        <figure id="pointer-target-stacked-links">
-            <img src="img/pointer-target-example7-stacked-links.png" alt="A bulleted list of stacked links with sufficient and insufficient offset (see legend below)" />
-          <figcaption>The two examples show lists of stacked links as used in navigation, for example in a list of contents of a longer text page. In both cases, the font size is 14px. In the first example, the line height is 25.8px, exceeding the requirement of 24 CSS pixels. In the second example, the stacked links have a line height of just 16.8px. They do not meet the vertical offset requirement of 24 CSS pixels.</figcaption>
-      </figure>
+
 
         <p>Users with different disabilities have different needs for control sizes. It can be beneficial to provide an option to increase the active target area without increasing the visible target size. Another option is to provide a mechanism to control the density of layout and thereby change target size or spacing, or both. This can be beneficial for a wide range of users. For example, users with visual field loss may prefer a more condensed layout with smaller sized controls while users with other forms of low vision may prefer large controls.</p>
 
-        <p>The aim of this Success Criterion is that targets are ideally at least 24 by 24 CSS pixels. However, the offset exception allows for smaller targets if there is sufficient spacing around them. Authors should be aware that using smaller targets can have an impact on layouts, as the combined space of the smaller target and the required spacing around it will generally require an overall area that is larger than 24 by 24 CSS pixels.</p>
+        <p>The aim of this Success Criterion is that targets are ideally at least 24 by 24 CSS pixels. However, the offset exception allows for smaller targets if there is sufficient spacing around them. Authors should be aware that using smaller targets can have an impact on layouts, as the combined space of the smaller target and the required spacing around it will generally require an overall size that is larger than 24 by 24 CSS pixels.</p>
```

Comment:
```suggestion
        <p>The aim of this Success Criterion is that targets are ideally at least 24 by 24 CSS pixels. However, the Spacing exception allows for smaller targets if there is sufficient spacing around them. Authors should be aware that using smaller targets can have an impact on layouts, as the combined space of the smaller target and the required spacing around it will generally require an overall size that is larger than 24 by 24 CSS pixels.</p>
```